x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/sample_logs_input.test.tsx

```diff
@@ -27,6 +27,119 @@ const changeFile = async (input: HTMLElement, file: File) => {
   });
 };
 
+const simpleNDJSON = `{"message":"test message 1"}\n{"message":"test message 2"}`;
+const multilineNDJSON = `{"message":"test message 1"}\n\n{\n  "message":\n  "test message 2"\n}\n\n`;
+const splitNDJSON = simpleNDJSON.split('\n');
+const complexEventsJSON = `{"events":[\n{"message":"test message 1"},\n{"message":"test message 2"}\n]}`;
+const nonIdentifierLikeKeyInJSON = `{"1event":[\n{"message":"test message 1"},\n{"message":"test message 2"}\n]}`;
+
+describe('parseNDJSON', () => {
```

Comment:
As we specifically catch `parseNDJSONError` and `parseJSONError` errors in `parseLogsContent`, we should add tests to verify that `parseNDJSON` and `parseJSON` are throwing their respective errors.

Reply:
I am not sure what you are referring to. There are three tests with ` expect( ... .toThrow())` below; do you mean some other tests?

Reply:
We test that the function throws an error (any kind), not just those specifically caught by the calling function. 
I would specify `... .toThrow(parseNDJSONError)`

Reply:
But we catch all error – this is a catch-all

Reply:
`parseNDJSONError` is just a variable name; if it was type it would start with an uppercase letter.

Reply:
arg yeah my mistake!